libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -498,7 +498,8 @@ bool ASTJsonConverter::visit(EventDefinition const& _node)
 		make_pair("nameLocation", sourceLocationToString(_node.nameLocation())),
 		make_pair("documentation", _node.documentation() ? toJson(*_node.documentation()) : Json::nullValue),
 		make_pair("parameters", toJson(_node.parameterList())),
-		make_pair("anonymous", _node.isAnonymous())
+		make_pair("anonymous", _node.isAnonymous()),
+		make_pair("eventSelector", _node.functionType(true)->externalIdentifierHex())
```

Comment:
Is this really what we want to output? Compare with https://github.com/ethereum/solidity/pull/12548.